packages/contracts-bridge/contracts/BridgeRouter.sol

```diff
@@ -290,15 +324,13 @@ contract BridgeRouter is Version0, Router {
             IBridgeToken(_token).setDetailsHash(_action.detailsHash());
         }
         // dust the recipient if appropriate
-        _dust(_recipient);
+        if (_isConnext) {
```

Comment:
thinking through how to reduce these `if/else` branches in the code and reduce sub-functions to a unified path of functionality. we _could_  

- pull `_dust` and `reconcile` calls up into the higher-level `handle` function (after) calling this function)
- remove references to `externalId` from this fn
- pass `recipient` as a param to this function; in higher-up fn, pass `_action.evmRecipient` or `address(connext)` as `recipient` depending on message type